src/bgfx.cpp

```diff
@@ -344,9 +344,13 @@ namespace bgfx
 		{
 			BGFX_FATAL(true
 				&& g_platformData.ndt     == _data.ndt
+#if !BX_PLATFORM_ANDROID
```

Comment:
Should be this instead:
`&& (_platformData.nwh     == _data.nwh || BX_ENABLED(BX_PLATFORM_ANDROID) )`